### PR TITLE
ADiaz-Drift

### PR DESCRIPTION
**Pull request by AdrianaRoseDiaz**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
New Pack to support Drift Integration

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [x] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Drift/Integrations/Drift/Drift.py [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+''' CONSTANTS '''
+API_ENDPOINT = 'https://driftapi.com'
+
+
+class Client(BaseClient):
+
+    def __init__(self, base_url, access_token, verify, proxy, email):
+        super().__init__(base_url=base_url, verify=verify, proxy=proxy)
+        self.access_token = access_token
+        self.email = email
```
Comment (MosheEichler) [inline Packs/Drift/Integrations/Drift/Drift.py:13]: Why do we need the email as a class variable?

File: Packs/Drift/Integrations/Drift/Drift.py [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+    def test(self):
+        url_suffix = '/users/list'
+        params = {'access_token': self.access_token}
+        self._http_request('GET', url_suffix=url_suffix, params=params)
+
+    def post_contact(self, email: str = None):
```
Comment (MosheEichler) [inline Packs/Drift/Integrations/Drift/Drift.py:20]:
```suggestion
    def post_contact(self, email: dict = None):
```

File: Packs/Drift/Integrations/Drift/Drift.py [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+    command = demisto.command()
+
+    commands = {
+        'drift-post-contact': post_contact_command,
+        'drift-get-contact': get_contact_command,
+        'drift-update-contact': patch_contact_command,
+        'drift-delete-contact': delete_contact_command
+    }
+
+    LOG(f'Command being called is {command}')
```
Comment (MosheEichler) [inline Packs/Drift/Integrations/Drift/Drift.py:134]:
```suggestion
    demisto.debug(f'Command being called is {command}')
```

File: Packs/Drift/Integrations/Drift/Drift.py [hunk 4]
```diff
@@ -0,0 +1,156 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+''' CONSTANTS '''
+API_ENDPOINT = 'https://driftapi.com'
```
[the remark this answers is not in the document]
Comment (AdrianaRoseDiaz) [reply Packs/Drift/Integrations/Drift/Drift.py:5]: Yep, the Drif api is very simple and this endpoint is stagnant, no customization.

File: Packs/Drift/Integrations/Drift/Drift.py [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+    return_results('ok')
+
+
+def post_contact_command(client, args):
+    email = {
+        'attributes': {
+            'email': args.get('email')
+        }
+    }
+    res = client.post_contact(email=email)
```
Comment (MosheEichler) [inline Packs/Drift/Integrations/Drift/Drift.py:70]: `email` argument is expected to be a str but a dict was sent

File: Packs/Drift/README.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for Drift.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | Access Token | API Access Token | True |
+    | Use system proxy settings |  | False |
+    | Trust any certificate (not secure) |  | False |
+    | Fetch indicators |  | False |
```
Comment (MosheEichler) [inline Packs/Drift/README.md:15]:
```suggestion
```
Please remove

**Review by MosheEichler — COMMENTED** [review]
Please see the final fix And we are ready to go :)

**Pull request merged**